source/app-hosting/index.rst

```diff
@@ -53,13 +53,15 @@ The SRCF uses Apache to serve websites so if you need to run a backend web app,
 Using UNIX sockets
 ^^^^^^^^^^^^^^^^^^
 
-You will need to configure your application to use a UNIX socket. You should make sure the socket is only accessible to you, either by using appropriate file modes or by picking a path that is only accessible to you such as ``/home/crsid/myapp/web.sock``. Then add the following to your ``.htaccess`` file, replacing as necessary::
+You will need to configure your application to use a UNIX socket. The socket should only be accessible to you, which can be done by picking a path in your area (e.g. ``/home/ab123/myapp/web.sock``) or by using appropriate file modes.
```

Comment:
"in your area", while intuitive, is I think too ambiguous.  Could we say something like "under your home directory" (or home folder, as that's a more familiar term) instead?